mods/ra/maps/allies-05a/AI.lua

```diff
@@ -61,8 +61,8 @@ ProtectHarvester = function(unit)
 
 		local Guards = SetupAttackGroup()
 		Utils.Do(Guards, function(unit)
-			if attacker.Location then
-				unit.AttackMove(attacker.Location)
+			if not self.IsDead then
```

Comment:
`SetupAttackGroup` should already do that: https://github.com/abcdefg30/OpenRA/blob/protectharvester/mods/ra/maps/allies-05a/AI.lua#L28